frontend/src/components/SearchTable.js

```diff
@@ -20,38 +20,40 @@ export default function SearchTable(props) {
       ) : (
         ""
       )}
-      <div className="border mb-4">
-        <Table
-          columns={props.columns}
-          rows={props.rows}
-          selectedRowIndex={3}
-          onColumnSort={(cName, cIndex) => {
-            console.log("cname is ", cName);
-            console.log("cIndex is ", cIndex);
-          }}
-          onRowSelect={(rName, rIndex) => {
-            console.log(rName, rIndex);
-            setShowModal(true);
-          }}
-        />
-      </div>
-
-      <Row className="text-center pagination ml-3">
+      <Row>
+        <Col>
+          <Table
+            columns={props.columns}
+            rows={props.rows}
+            selectedRowIndex={3}
+            onColumnSort={(cName, cIndex) => {
+              console.log("cname is ", cName);
+              console.log("cIndex is ", cIndex);
+            }}
+            onRowSelect={(rName, rIndex) => {
+              console.log(rName, rIndex);
+              setShowModal(true);
+            }}
+          />
+        </Col>
+      </Row>
+      <Row className="text-center pagination">
         <Col sm={1} />
         <Col sm={9}>
           <Pagination
-            numberOfPages={
-              Math.ceil(props.pagination.totalNumberOfResults /
-              props.pagination.results_per_page)
-            }
+            numberOfPages={Math.ceil(
+              props.pagination.totalNumberOfResults /
+                props.pagination.results_per_page
+            )}
             currentPageIndex={props.pagination.page}
             resultsIndexStart={
               props.pagination.page * props.pagination.results_per_page
             }
             resultsIndexEnd={
-              (props.pagination.page + 1) * (props.pagination.results_per_page) - 1
+              (props.pagination.page + 1) * props.pagination.results_per_page -
+              1
             }
-            onSetPage={(index) => props.pagination.page = index}
+            onSetPage={(index) => (props.pagination.page = index)}
```

Comment:
Not sure what is going on here.  You are setting the prop to index?  Should you be setting state?

Reply:
I'm not exactly sure- maybe it's intended as a Redux update? I can look into this if you'd like but I think we're reworking the pagination which will hopefully fix this